### PR TITLE
Mothership: Enable proxyd to devnet(include srdg), change op-geth registry

### PR DESCRIPTION
**Pull request by ost006**
- Add proxyd to the Mothership devnet.
- The Bundler now calls RPCs through the proxy.
- Change the registry to use a customized version of op-geth for interacting with the bundler related to EIP-7212.
- Proxyd only supports the x86_64 architecture.

### CODE REVIEW

File: charts/mothership/values.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ node:
   count: 1
   loadBalancerExternal: false
   opGeth:
-    image: &op-geth-image stackupwallet/op-geth:v1.101315.2
+    image: &op-geth-image ghcr.io/planetarium/op-geth:latest
```
Comment (eseiker) [inline charts/mothership/values.yaml:24]:
```suggestion
    image: &op-geth-image ghcr.io/planetarium/op-geth:v1.101315.2
```
I've just tagged it
https://github.com/planetarium/op-geth/pkgs/container/op-geth/238438298?tag=v1.101315.2

File: charts/mothership/values.yaml [hunk 2]
```diff
@@ -65,7 +65,7 @@ bundler:
   enabled: false
   image: stackupwallet/stackup-bundler:v0.6.47
   loadBalancerExternal: false
-  gethRpc: http://node-1:8545
+  gethRpc: http://proxy:8080
```
Comment (eseiker) [inline charts/mothership/values.yaml:68]:
```suggestion
  gethRpc: http://proxyd:8080
```

**Review by eseiker — COMMENTED** [review]
(+ It'd be good to use proxyd for other services such as blockscout)

**Review by eseiker — APPROVED** [review]

File: charts/mothership/values.yaml (hunk 2 above)
Comment (ost006) [reply charts/mothership/values.yaml]: you're right. it should be `proxyd`

Comment (ost006) [reply charts/mothership/values.yaml]: Using `proxyd` to `blockscout` is currently being tested and may be applied in the future.

**Pull request merged**